scripts/create_ts_for_integration_tests.py

```diff
@@ -222,7 +270,11 @@ def create_dense_time_series(ts_api):
 
 if __name__ == "__main__":
     # To avoid accidental runs, please provide a valid config to CogniteClient:
-    client = CogniteClient(...)
+    from scripts import local_client
+
+    client = local_client.get_interactive()
+
     delete_all_time_series(client.time_series)
-    create_dense_time_series(client.time_series)
+    ts_lst, df_lst = create_dense_time_series()
+    create_if_not_exists(client.time_series, ts_lst, df_lst)
     create_edge_case_time_series(client.time_series)
```

Comment:
What about `create_edge_case_time_series` (w.r.t. "check if exist"?)

Reply:
Laziness only commented it out. The create if not exist was to avoid deleting everything, when I has some windows issues with the creation. (Forgot that windows like np.float32). Fixing it.